protos/Supervisor.proto

```
@@ -0,0 +1,109 @@
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+syntax = "proto3";
+
+package crane.grpc.supervisor;
+option go_package = "CraneFrontEnd/generated/protos/supervisor";
+
+import "PublicDefs.proto";
```

Buf (1.47.2)

import "PublicDefs.proto": file does not exist

(COMPILE)